utils/hono-influxdb-connector/src/main/java/org/eclipse/kuksa/honoConnector/HonoInfluxConnection.java

```diff
@@ -93,7 +98,12 @@ public HonoInfluxConnection(final ClientConfigProperties honoConfig,
      */
 	public void connectToHono() {
         try {
-            Future<HonoClient> future = honoClient.connect(options);
+            Future<HonoClient> future = honoClient.connect(options, unused -> {
+            	// The disconnect handler is invoked when the TCP connection is lost for any
+            	// reason e.g. a network outage or when Hono is restarted.
+            	LOGGER.info("AMQP connection was disconnected");
```

Comment:
What do you think about adding a comment, that the AMQP connection will be reconnected? Like "AMQP connection was disconnected and try to reconnect now"

Reply:
The reconnect() has logging that infos about the reconnect attempt.